vpr/src/draw/draw_types.h

```diff
@@ -143,6 +143,11 @@ typedef struct {
     bool node_highlighted;
 } t_draw_rr_node;
 
+typedef struct {
```

Comment:
Make a doxygen comment for this.
Can @brief to say what the structure is for.
Can use ///@brief above a variable to say what it is, or could just put it in a block comment all together.